scripts/release/mule/sign/sign.sh

```diff
@@ -47,7 +55,7 @@ do
     gpg -u rpm@algorand.com --detach-sign "$file"
 done
 
-HASHFILE=hashes_${CHANNEL}_${OS_TYPE}_${ARCH_TYPE}_${VERSION}
+HASHFILE="hashes_${CHANNEL}_${OS_TYPE}_${ARCH_TYPE}_${VERSION}"
```

Comment:
Will this sign all of the packages for different architectures in tmp/node_dir, or just the ones matching the host computers cpu architecture?

Reply:
The latter.

Reply:
Cool, for this one we can run this for each os/arch we'll need. Later on maybe we can update this like the package scripts to iterate on all available files